Assets/Scripts/PlayerMovement.cs

```diff
@@ -31,19 +31,20 @@ void Update()
         //This is obviously not a great way of doing this.
         //This could be changed with the new input system but I just wanted to quickly test this.
 
-        if (Input.GetKeyDown(KeyCode.UpArrow))
+
+        if (Input.GetKeyDown(KeyCode.UpArrow) || Input.GetKeyDown(KeyCode.W) )
```

Comment:
Ideally this movement code should be refactored to use input axis instead of KeyCode. But this still works fine for only wasd and arrow keys.

Just a bit gross